app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -96,6 +100,45 @@ private void onNotFound() {
         }
     }
 
+
+    /**
+     * Helper method that will split events into upcoming / past
+     * and add to the adapter of this list fragment
+     *
+     * @param events
+     *      The events list to be added
+     */
+    private void splitEventsAndAddToAdapter(Collection<? extends SimpleEvent> events) {
+        Pair<List<SimpleEvent>, List<SimpleEvent>> pair = splitEventsList(events);
+        mAdapter.addAll(pair.first);
+        mAdapter.addAll(pair.second);
```

Comment:
But first it's split and that added to the same adapter without any distinction. I'm just wondering of the same could be achieved by a comparator.